tutorials/plot_stats.py

```diff
@@ -21,32 +21,32 @@
 
 # sphinx_gallery_thumbnail_number = 3
 
-# import auxiliary libraries
+# Import auxiliary libraries
 import numpy as np
 import matplotlib.pyplot as plt
 
 # Import statistics-related functions
-from spiketools.stats.generators import poisson_train
+from spiketools.stats.generators import poisson_generator
 from spiketools.stats.shuffle import (shuffle_isis, shuffle_bins, shuffle_poisson,
                                       shuffle_circular)
-from spiketools.stats.permutations import compute_empirical_pvalue, zscore_to_surrogates
+from spiketools.stats.permutations import compute_surrogate_stats
 
 # Import plot function
 from spiketools.plts.trials import plot_rasters
 from spiketools.plts.stats import plot_surrogates
 
 # Import measures & utilities
-from spiketools.utils.spikes import restrict_range
-from spiketools.measures.measures import compute_spike_rate
+from spiketools.utils.data import restrict_range
+from spiketools.measures.measures import compute_firing_rate
 
 ###################################################################################################
 
 # Generate spike times for spikes at 10Hz for 100 seconds
-poisson_generator = poisson_train(10, 100)
-# get spike in seconds
-spikes_s = np.array([spike for spike in poisson_generator])
-# convert to milliseconds
-spikes_ms = np.unique((spikes_s*1000).astype(int))
+poisson_gen = poisson_generator(10, 100)
+# get spike times in seconds
+spikes_s = np.array([spike for spike in poisson_gen])
+# make sure there are not multiple spikes within the same millisecond
```

Comment:
This is equivalent to adding a refractory period of 1 millisecond, which can also be done directly in `sim_spiketimes`. 

Combining both comments above, I think the simulation here could be done like this:
`spikes = sim_spiketimes(10, 100, 'poisson', refractory=0.001)`

Reply:
Also, since we know default to spikes in seconds, i think you can call this `spikes`, rather than `spikes_s`.